caravel/views.py

```diff
@@ -1238,6 +1237,48 @@ def testconn(self):
                 status=500,
                 mimetype="application/json")
 
+    @api
+    @has_access_api
+    @expose("/warm_up_cache/", methods=['GET', 'POST'])
+    def warm_up_cache(self):
+        """Warms up the cache for the slice or table."""
+        slices = None
+        s = db.session()
```

Comment:
`s` is commonly used for short lived strings, I'd go with `session` instead to avoid confusion